.travis.yml

```diff
@@ -1,73 +1,4 @@
-language: php
+version: ~> 1.0
 
-dist: trusty
-
-before_install:
-  - sudo apt-get update
-  - sudo apt-get install chromium-chromedriver
-
-env:
-  global:
-     - TRAVIS_NODE_VERSION="10"
-     - DISPLAY=":99"
-     - XVFBARGS=":99 -ac -screen 0 1024x768x16"
-     - SS_BASE_URL="http://localhost:8080/"
-     - SS_ENVIRONMENT_TYPE="dev"
-
-matrix:
-  include:
-    - php: '7.1'
-      env: DB=MYSQL RECIPE_VERSION=4.3.x-dev PDO=1 PHPUNIT_COVERAGE_TEST=1
-    - php: '7.2'
-      env: DB=MYSQL RECIPE_VERSION=4.3.x-dev PHPUNIT_TEST=1
-    - php: '7.3'
-      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev PHPUNIT_TEST=1
-    - php: '7.2'
-      env: DB=MYSQL RECIPE_VERSION=4.5.x-dev PHPUNIT_TEST=1
-    - php: '7.3'
-      env: DB=MYSQL RECIPE_VERSION=4.x-dev PHPUNIT_TEST=1 PHPCS_TEST=1
-    - php: '7.3'
-      env: DB=MYSQL RECIPE_VERSION=4.x-dev NPM_TEST=1
-
-before_script:
-  # Extra $PATH
-  - export PATH=/usr/lib/chromium-browser/:$PATH
-
-  # Init PHP
-  - phpenv rehash
-  - phpenv config-rm xdebug.ini
-  - export PATH=~/.composer/vendor/bin:$PATH
-  - echo 'memory_limit = 2G' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-
-  # Install composer
-  - composer validate
-  - if [[ $DB == PGSQL ]]; then composer require --prefer-dist --no-update silverstripe/postgresql 2.1.x-dev; fi
-  - composer require --no-update silverstripe/recipe-testing:^1 silverstripe/recipe-cms:"$RECIPE_VERSION"
-  - composer install --prefer-source
-
-  # Behat bootstrapping
-  - if [[ $BEHAT_TEST ]]; then mkdir artifacts; fi
-  - if [[ $BEHAT_TEST ]]; then cp composer.lock artifacts/; fi
-  - if [[ $BEHAT_TEST ]]; then sh -e /etc/init.d/xvfb start; sleep 3; fi
-  - if [[ $BEHAT_TEST ]]; then (chromedriver > artifacts/chromedriver.log 2>&1 &); fi
-  - if [[ $BEHAT_TEST ]]; then (vendor/bin/serve --bootstrap-file vendor/silverstripe/cms/tests/behat/serve-bootstrap.php &> artifacts/serve.log &); fi
-
-  # Install NPM dependencies
-  - if [[ $NPM_TEST ]]; then nvm install $TRAVIS_NODE_VERSION && nvm use $TRAVIS_NODE_VERSION && npm install -g yarn && yarn install --network-concurrency 1 && (cd vendor/silverstripe/admin && yarn install --network-concurrency 1) && yarn run build; fi
-
-script:
-  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit; fi
-  - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs src tests; fi
-  - if [[ $NPM_TEST ]]; then git diff-files --quiet -w --relative=client; fi
-  - if [[ $NPM_TEST ]]; then git diff --name-status --relative=client; fi
-  - if [[ $NPM_TEST ]]; then yarn run coverage; fi
-  - if [[ $NPM_TEST ]]; then yarn run lint; fi
-  - if [[ $BEHAT_TEST ]]; then vendor/bin/behat @security-extensions; fi
```

Comment:
I'm wondering, did we lose these behat tests somewhere along the road? Might be worth investigating before we completely get rid of it from the config

Reply:
I've looked through previous releases https://github.com/silverstripe/silverstripe-security-extensions/releases - I don't think these behat tests ever existed.  I suspect there was an intention to write some at some stage, though they were never actually written

Reply:
ok, let's ship it then